vhs-frontend/package.json

```diff
@@ -59,5 +62,9 @@
       "last 1 safari version"
     ]
   },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx}": "eslint --fix",
+    "*.{js,jsx,ts,tsx,css,md,json}": "prettier --write"
```

Comment:
You can make this shorter. For starters, use `yarn lint:fix` and `yarn format` scripts to run these since you already have them defined under `scripts`.

Also, I don't personally like to use `eslint --fix` with the fix option here because then you don't have full control over the code that actually gets commited. In case you accidentally change some rules or you do it in an improper way, this will cause your code to be commited in a different style than expected, potentially causing problems you will be unaware of.

I think it's better to just lint the code and then fix the reported problems yourself, usually there are very few and you get full control over the final commited code.

You don't have to change this if you don't want to, but it's important to discuss it. You can roll with this and see how it will work out for you. :)